code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -14,6 +14,51 @@
 	set desc = "Evolve into a higher form."
 	set category = "Alien"
 
+	do_evolve()
+
+
+/mob/living/carbon/xenomorph/verb/regress()
+	set name = "Regress"
+	set desc = "Regress into a lower form."
+	set category = "Alien"
+	
+	var/tiers_to_pick_from
+	switch(tier)
+		if(XENO_TIER_ZERO, XENO_TIER_FOUR)
+			to_chat(src, "<span class='warning'>Your tier does not allow you to regress.</span>")
+			return
+		if(XENO_TIER_ONE)
+			tiers_to_pick_from = list(/mob/living/carbon/xenomorph/larva)
+		if(XENO_TIER_TWO)
+			tiers_to_pick_from = GLOB.xeno_types_tier_one
+		if(XENO_TIER_THREE)
+			if(isxenoshrike(src))
+				tiers_to_pick_from = GLOB.xeno_types_tier_one
+			else
+				tiers_to_pick_from = GLOB.xeno_types_tier_two
+		else
+			CRASH("side_evolve() called without a valid tier")
```

Comment:
*cries*

Reply:
Want me to make a PR to remove every single instance of `CRASH`? Not sure what that'd accomplish.

Reply:
No? I'm just complaining about a pet peeve while approving the PR. You're free to resolve the conversation and ignore it.